gulpfile.js

```diff
@@ -57,7 +57,7 @@ gulp.task('dev', (done) => {
 gulp.task('watch', function() {
     browserSync.init({
         server: {
-            baseDir: "./www",
+            baseDir: "./src",
```

Comment:
OK, sorry I can remove that – it wasn't working for me locally, which is why I changed it. I guess I just hadn't pulled in the latest changes. Will revert now.